vector/src/main/java/im/vector/app/features/settings/devices/v2/list/CurrentSessionView.kt

```diff
@@ -39,21 +39,14 @@ class CurrentSessionView @JvmOverloads constructor(
         views = ViewCurrentSessionBinding.bind(this)
     }
 
-    fun update(accountCrossSigningIsTrusted: Boolean, legacyMode: Boolean) {
-        renderDeviceType()
-        renderVerificationStatus(accountCrossSigningIsTrusted, legacyMode)
+    fun update(currentDeviceInfo: DeviceFullInfo) {
```

Comment:
In custom views, it seems we used `render()` to name the method to refresh the views. Maybe we should align to that?

Reply:
Done.